media-video/ffdiaporama/ffdiaporama-2.2_pre20140701-r2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit xdg-utils gnome2-utils qmake-utils
+
+MY_PV=${PV%_pre20140701}
+BIN_PV=${MY_PV}.devel.2014.0701
+RSC_PV=${MY_PV}.devel.2014.0503
+TMT_PV=1.0.2014.0125
+DESCRIPTION="Movie creator from photos and video clips"
+HOMEPAGE="http://ffdiaporama.tuxfamily.org"
+SRC_URI="http://download.tuxfamily.org/${PN}/Packages/Devel/${PN}_bin_${BIN_PV}.tar.gz
+	http://download.tuxfamily.org/${PN}/Packages/Devel/${PN}_rsc_${RSC_PV}.tar.gz
+	https://dev.gentoo.org/~jstein/dist/ffdiaporama-2.2-libav11.patch
+	https://dev.gentoo.org/~jstein/dist/ffdiaporama-2.2-ffmpeg-3.0.patch
+	https://dev.gentoo.org/~jstein/dist/ffdiaporama-2.2-ffmpeg-4.0.patch
+	texturemate? ( http://download.tuxfamily.org/${PN}/Packages/Stable/${PN}_texturemate_${TMT_PV}.tar.gz )"
+
+LICENSE="GPL-2
+	texturemate? ( CC-BY-3.0 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libav openclipart texturemate"
+
+RDEPEND="
+	media-gfx/exiv2
```

Comment:
exiv2 is likely missing a `:=` slot operator there.
Also: Please sort those dependencies lexicographically, and optional ones below the unconditional deps.